cpp/src/barretenberg/ecc/scalar_multiplication/scalar_multiplication.test.cpp

```diff
@@ -917,7 +917,7 @@ TYPED_TEST(SRSIO, PippengerUnsafeShortInputs)
 
     Fr* scalars = (Fr*)aligned_alloc(32, sizeof(Fr) * num_points);
 
-    AffineElement* points = (AffineElement*)aligned_alloc(32, sizeof(AffineElement) * num_points * 2 + 1);
```

Comment:
Any reason this can't just be a points = new AffineElement[num_points * 2 + 1]?

Reply:
We need them aligned. From what I understand by default new doesn't do that and we absolutely need that for pippenger